styles.css

```diff
@@ -0,0 +1 @@
+h1 { color: blue; }
```

Comment:
- [OPTIONAL] In my opinion you should use a class as recommended by the [project requirements](https://github.com/danielmatama/hello-world/pull/1#pullrequestreview-1197757428) to make it look more professional and clean